go.mod

```diff
@@ -99,3 +99,5 @@ replace (
 	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.19.0
 	k8s.io/sample-controller => k8s.io/sample-controller v0.19.0
 )
+
+replace github.com/code-ready/machine => github.com/guillaumerose/machine v0.0.0-20201224094349-747e5fa2ff83
```

Comment:
what still remains there?

Reply:
not much https://github.com/code-ready/machine/pull/54

the RPC, driver interface and some common structs. Few lines.